packages/databases-collections/src/components/collection-fields/time-series-fields.jsx

```diff
@@ -1,11 +1,22 @@
 import React, { useCallback } from 'react';
 import PropTypes from 'prop-types';
 import { Select, Option, TextInput } from '@mongodb-js/compass-components';
+import { spacing, css } from '@mongodb-js/compass-components';
 
 import FieldSet from '../field-set/field-set';
 import CollapsibleFieldSet from '../collapsible-field-set/collapsible-field-set';
 
-import styles from './time-series-fields.module.less';
+const optionsInputStyles = css({
+  paddingLeft: spacing[4]
```

Comment:
Instead of having each input with a padding, would it make sense to have the `CollapsibleFieldSet` provide this padding so the children of the component are already shifted to the side?